prophy/composite.py

```diff
@@ -519,18 +529,18 @@ def __str__(self):
         value = getattr(self, name)
         return field_to_string(name, tp, value)
 
-    def fields(self, prefix=''):
-        out = []
-        name, tp, _, _, _ = self._discriminated
-        value = getattr(self, name, None)
-        return field_to_list(prefix, name, tp, value)
 
-    def values(self, prefix=''):
-        out = {}
-        name, tp, _, _, _ = self._discriminated
-        value = getattr(self, name, None)
-        out.update(field_to_dict(prefix, name, tp, value))
-        return out
+    def get_discriminated(self):
+        name, tp, _, _, _  =  self._discriminated
+        return Descriptor(name,tp,get_kind(tp))
```

Comment:
In the sake of consistency with the rest of library, use "x, y" style instead of "x,y" when separating argument list.